spring-cloud-vault-spring-connector/src/main/java/io/pivotal/spring/cloud/vault/config/java/VaultServiceInfoPropertySourceAdapter.java

```diff
@@ -18,21 +18,28 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import io.pivotal.spring.cloud.vault.service.common.VaultServiceInfo;
-
 import org.springframework.core.env.MapPropertySource;
 import org.springframework.core.env.PropertySource;
 
+import io.pivotal.spring.cloud.vault.service.common.VaultServiceInfo;
```

Comment:
Please do not reformat import.

Reply:
I am really lost which formatter to use. I have end up reformatting manually, this is why I have ended up with multiple commits. This one I have missed. Please provide formatter which I can use moving forward.

Reply:
We're using Spring Vault's formatter and import-order config, see https://github.com/spring-projects/spring-vault/tree/master/etc/ide.